services/alerts/src/index.ts

```diff
@@ -1,3 +1,4 @@
 export { AlertsProvider } from './AlertsProvider'
 export { useAlerts } from './useAlerts'
 export { useAlert } from './useAlert'
+export { AlertsManagerContext } from './AlertsManagerContext'
```

Comment:
this is now being exported so that the PluginSender can access `add` from the parent (app) and pass it to the plugin, so that the alerts can then be added to the app's AlertsManager rather than the plugin's